app/src/main/res/values/strings.xml

```diff
@@ -128,6 +130,10 @@
     <string name="unsupported_url_dialog_message">Could not recognize the URL. Open with another app?</string>
     <string name="default_content_country_title">Default content country</string>
     <string name="content_language_title">Default content language</string>
+    <plurals name="replies">
+        <item quantity="one">%s Reply</item>
+        <item quantity="other">%s Replies</item>
```

Comment:
> and thus does not need to encourage participation

I don't see how capitalisation or the lack of it could encourage or discourage participation.

But yes, the words should be in lowercase for both scenarios. "1 reply" and "10 replies".

Reply:
I think fully capitalized is because it's a button

Reply:
Oh, you're saying Android does this automatically?

Reply:
> Oh, you're saying Android does this automatically?

No, it's me manually code, I just think maybe a button text should be fully capitalized? https://github.com/TeamNewPipe/NewPipe/blob/f29c941241945aab4a876b51bac7e6347fd4fe5c/app/src/main/res/layout/list_comments_item.xml#L125

Reply:
That might be the case normally, but it's not looking good hear. It's somewhat distracting from the actual comments, especially the blue colour.

Try using the regular text colour, and maybe use border/fill colour to highlight that it is a button?

Reply:
I think using blue is ok.

Reply:
I already love it as it is, blue and not capitalized. Blue is good as you can see very well that there are replies for those comments. I agree that capitalized does not look good.

Reply:
So... let's see a screenshot of blue but not capitalised?

Reply:
![Screenshot_20221101-132627_NewPipe Debug](https://user-images.githubusercontent.com/32761048/199165292-9355a2f9-5e3a-4c5a-a373-75702f5b7e72.png)

Reply:
Ah! Perfect! It looks great now.

Reply:
I'll change it like [material 3 say](https://m3.material.io/components/buttons/guidelines#28a1ae9b-9576-44a8-8d2e-8231a0218d20)